src/lavinmq/shovel/shovel_store.cr

```diff
@@ -40,12 +40,6 @@ module LavinMQ
       end
     end
 
-    def each(&)
```

Comment:
we currently use `each_value` for shovels, instead of its own `each`
why remove `forward_missing_to` and exchange `each_value` for `each` instead of just removing `each`?